src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -140,29 +140,30 @@ private void ProcessType(INamedTypeSymbol typeSymbol)
 					}
 
 					var builder = new IndentedStringBuilder();
-					builder.AppendLineIndented("// <auto-generated>");
-					builder.AppendLineIndented("// ******************************************************************");
-					builder.AppendLineIndented("// This file has been generated by Uno.UI (DependencyObjectGenerator)");
-					builder.AppendLineIndented("// ******************************************************************");
-					builder.AppendLineIndented("// </auto-generated>");
-					builder.AppendLine();
-					builder.AppendLineIndented("#pragma warning disable 1591 // Ignore missing XML comment warnings");
-					builder.AppendLineIndented($"using System;");
-					builder.AppendLineIndented($"using System.Linq;");
-					builder.AppendLineIndented($"using System.Collections.Generic;");
-					builder.AppendLineIndented($"using System.Collections;");
-					builder.AppendLineIndented($"using System.Diagnostics.CodeAnalysis;");
-					builder.AppendLineIndented($"using Uno.Disposables;");
-					builder.AppendLineIndented($"using System.Runtime.CompilerServices;");
-					builder.AppendLineIndented($"using Uno.UI;");
-					builder.AppendLineIndented($"using Uno.UI.Controls;");
-					builder.AppendLineIndented($"using Uno.UI.DataBinding;");
-					builder.AppendLineIndented($"using Windows.UI.Xaml;");
-					builder.AppendLineIndented($"using Windows.UI.Xaml.Data;");
-					builder.AppendLineIndented($"using Uno.Diagnostics.Eventing;");
-					builder.AppendLineIndented("#if __MACOS__");
-					builder.AppendLineIndented("using AppKit;");
-					builder.AppendLineIndented("#endif");
+					builder.Append(@"// <auto-generated>
+// ******************************************************************
```

Comment:
Not familiar with this project setup, but I see it's using the 4.4 toolset, doesn't that mean we can also use raw multiline string literals here then? Would make this bit a bit nicer, with proper indentation in the code 🙂

Reply:
We're using C# 10:

https://github.com/unoplatform/uno/blob/18f9d47cf7b944820a49dec723ebc574e1cd936f/src/Directory.Build.props#L51

I think it should be no problem to update to C# 11. I'll do that in a separate PR.